content/docs/setup/kubernetes/helm-install.md

```diff
@@ -32,13 +32,13 @@ recommended.
       (requires Kubernetes >=1.9.0):
 
         ```command
-        $ helm template @install/kubernetes/helm/istio@ --name istio --namespace istio-system > $HOME/istio.yaml
+        $ helm template @install/kubernetes/helm/istio@ --name istio --namespace istio-system --set global.tag=0.8.0 > $HOME/istio.yaml
```

Comment:
@geeknoid should this be shipped with 0.8?

Reply:
Are you asking whether to include this doc change in the release-0.8 branch so it goes to istio.io right away?

If so, then it probably makes sense to merge this change into release-0.8.

Reply:
@geeknoid I think this should only goes to 0.8 release but not master. In master, we should probably use the latest branch.

Reply:
created a new PR https://github.com/istio/istio.github.io/pull/1538

Reply:
The default tag in 0.8.0 values.yaml file is 0.8.0 so I'm not sure this makes sense against 0.8.0 branch of docs.

There may be other use cases that I don't understand.  Could you expand on the problem your trying to solve?

Thanks
-steve

Reply:
+1 to @sdake , Yes, 0.8 release already using `0.8.0` as default tag, so seems we do not need this PR.